<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="FullNameLine">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="FullNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TestMapBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TestMap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TestVectorBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TestVector</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MaleRadioBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FemaleRadioBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="unknownRadioBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showGenderBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowGender</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GenderLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="MorningCheckBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Buổi sáng</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AfternoonCheckBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Buổi chiều</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EveningCheckBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Buổi tối</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowFavoriteBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowFavorite</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShowPeriodLineEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
